.github/workflows/pr-preview.yml

```diff
@@ -26,6 +26,14 @@ jobs:
           aws-role-arn: ${{ secrets.AWS_ROLE_ARN }}
           aws-region: us-west-2
 
+      - name: Cache Astro cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            alchemy-web/node_modules/.astro
+            alchemy-web/.astro
+          key: astro-pr-cache-${{ github.event.pull_request.number }}
```

Comment:
Will this work on main?

Reply:
I don't think I understand the question. This workflow only runs on PRs so it won't run on main. I have a different cache key in publish.yml which will work on main fine.